hack/boilerplate/boilerplate.go.txt

```diff
@@ -1,5 +1,5 @@
 /*
-Copyright (C) YEAR Red Hat, Inc.
+Copyright YEAR The Kubernetes Authors All rights reserved.
```

Comment:
> The easiest way to skip the files with Kubernetes boilerplate 

-1

> The current implementations is very deeply tied up with one boilerplate for one type of file

I don't get it. There are boilerplate headers for the various file types, yes, but how is this all different than say the YEAR? Why don't you put COPYRIGHTHOLDER as a token in the various templates and treat this token the same way as the YEAR? See https://github.com/minishift/minishift/blob/8b500a691c670f5ee4de93a11b157de24f761fae/hack/boilerplate/boilerplate.py#L148-L148

Am I missing something here?